### PR TITLE
Remove Prebid Bidder Performance in Admin Frontend tools

### PR DESCRIPTION
**Pull request by deedeeh**
## What does this change?

This PR removes the Prebid Bidder Performance from Performance section in Commercial tools which is found in admin Frontend tools.

The Commercial team is tidying up the Commercial tools in admin page and this is the first step towards it.

Related PRs https://github.com/guardian/Prebid.js/pull/159 and https://github.com/guardian/platform/pull/1625

The merging order for the PRs as follows https://github.com/guardian/frontend/pull/27506#discussion_r1781452351

## Screenshots

| Before      | After      |
|-------------|------------|
| ![before][] | ![after][] |

[before]: https://github.com/user-attachments/assets/dbf21f42-e424-4f21-b32a-37bf1be3085d
[after]: https://github.com/user-attachments/assets/29b50c70-52e9-4e61-bfe2-ff2100fc88aa

## Checklist

- [x] Tested locally, and on CODE if necessary

### CODE REVIEW

**Review by emma-imber — APPROVED** [review]
Nice! I think we can also now get rid of the `dateLineCharts.scala.html` file as it looks like only the TeamKPIController was referencing it

**Review by arelra — APPROVED** [review]
✨

File: dev-build/conf/routes
Comment (deedeeh) [reply dev-build/conf/routes:277]: I am leaning towards removing the `/hb` endpoint from routes and dev-build routes and all related things to `PrebidAnalyticsController` and like you said the other two aspects could be a follow up.

Comment (arelra) [reply dev-build/conf/routes]: The only issue with removing `/hb` in this PR is that we'd need to remove the call to `/hb` from our prebid fork first so the call does not error in the clients. Probably better to remove from prebid first and then from Frontend.

Comment (deedeeh) [reply dev-build/conf/routes]: I can do that in Prebid and have a ready PR to merge then merge this PR after so I don't have to create two PRs in Frontend. What I am trying to understand right now the Platform repo router conf has to be removed when during the process I mentioned, any ideas?

Comment (arelra) [reply dev-build/conf/routes]: Sure thats fine as long you don't mind putting this PR on hold. I think Router changes changes can go anytime after we stop sending the `/hb` request from the client.

So first:
prebid (no more requests to `/hb`)

Then as there are no more requests to `/hb` we can do an the following in any order:
router
Frontend

**Pull request merged**
